app/components/Comments/CommentView.tsx

```diff
@@ -37,12 +37,7 @@ const CommentView = (props: Props) => {
   return (
     <div style={style}>
       <Title displayTitle={displayTitle} />
-      <Flex
-        gap="var(--spacing-md)"
-        style={{
-          flexDirection: newOnTop ? 'column-reverse' : 'column',
```

Comment:
This causes the comment field to be put at the bottom, which in my eyes is just unpractical

Reply:
I propose to keep it as it is, but rather move the commentform out of the flex component (or add a separate flex around the commenttree that can be reversed). If you check out e.g. https://abakus.no/bdb/4488 there are a lot of comments, and it would be useful to have the newest on top - with CommentForm above.

Update: I now noticed that there is actually some reversing going on in line 51 (`comments={newOnTop ? tree.reverse() : tree}`) when calling CommentTree, but that doesn't seem to work? as that should have made comments reverse twice..  
So remove either one of them but make sure the remaining one actually works

Reply:
hmm I assumed the `tree.reverse()` was enough. I'll look into it

Reply:
Lol my bad, `tree.reverse()` is definitely enough - ignore my previous comment(:

At first I thought the comments and the commentform were all in the same div, but they're evidently not

Reply:
So I agree with the changes you made